pkg/morph/client/constructor.go

```diff
@@ -158,6 +158,12 @@ func (c *Client) newCli(endpoint string) (*rpcclient.WSClient, *actor.Actor, *ne
 		return nil, nil, nil, fmt.Errorf("WS client creation: %w", err)
 	}
 
+	defer func() {
+		if err != nil {
+			cli.Close()
+		}
```

Comment:
Why don't we close it in any case (before or after `Init`)?

Reply:
cause i did not want to write `cli.Close()` in every `err != nil`, the only reason. we should close the cli even if `actor`
`gas` creation has failed because we will try to open a new ws conn anyway

i can change it to the repeated calls in every `if` branch if you want